lab/13-npm/classes.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
```

Comment:
`'use strict'` is implied automatically by the ES Module Loader. It's needed only for CommonJS modules.